edgecast/customer/customer_models.go

```diff
@@ -0,0 +1,388 @@
+package customer
+
+// Customer represents data for a Customer that is a child object of
+// the associated Partner. Customers contain Customer Users that are used to
+// login to my.edgecast.com (the MCC). For more information on user management,
+// please review the Web Services REST API (Partner) document available at the
+// below url:
+// https://partner.edgecast.com/support/default.aspx
+type Customer struct {
+	// A string that is reserved for future use.
+	AccountID string `json:"AccountId,omitempty"`
+
+	// A string identifying the street address for the customer.
+	Address1 string
```

Comment:
json deserialization format should be provided for all property.
Especially, if a response from a server omits one of properties defined here, it would throw parsing error(s).
Other solution is making it as pointer. Json parse in go would omit if pointer is nil.

Reply:
@changgyu-oh added to all properties.